robot_vision.py

```diff
@@ -0,0 +1,59 @@
+"""
+The main file to call that takes vision tracking information and sends to
+a network table.
+Use `java -jar support/OutlineViewer-1.0.1.jar` in terminal to create a local
+server, once you have everything installed (see README)
+"""
+from lib import config, tables, target_tracker, color_mask, util
+
+def run(cfg):
```

Comment:
Seems like it might be more clear if `run` took specific parameters:

    def run(channel, server, lower, upper)

And those values came from the configuration file and passed in ... of course, when we start to track multiple targets, perhaps that sort of *behavioral logic* would come from the configuration file, so ... maybe not.